repository-data/application/src/main/resources/hcm-content/taxonomies.yaml

```diff
@@ -0,0 +1,2836 @@
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: critical_care_science
+          locale.en.name: Critical care science
+        /cytopathology:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: cytopathology
+          locale.en.name: Cytopathology
+        /decontamination_and_sterile_services:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: decontamination_and_sterile_services
+          locale.en.name: Decontamination and sterile services
+        /gastrointestinal_physiology:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: gastrointestinal_physiology
+          locale.en.name: Gastrointestinal physiology
+        /genomic_counselling:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: genomic_counselling
+          locale.en.name: Genomic counselling
+        /genomics:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: genomics
+          locale.en.name: Genomics
+        /haematology__healthcare_scientist_:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: haematology__healthcare_scientist_
+          locale.en.name: Haematology (healthcare scientist)
+        /hearing_aid_dispenser:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: hearing_aid_dispenser
+          locale.en.name: Hearing aid dispenser
+        /histocompatibility_and_immunogenetics:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: histocompatibility_and_immunogenetics
+          locale.en.name: Histocompatibility and immunogenetics
+        /histopathology__healthcare_scientist_:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: histopathology__healthcare_scientist_
+          locale.en.name: Histopathology (healthcare scientist)
+        /imaging__ionising_:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: imaging__ionising_
+          locale.en.name: Imaging (ionising)
+        /imaging__non_ionising_:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: imaging__non_ionising_
+          locale.en.name: Imaging (non_ionising)
+        /infection_sciences:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: infection_sciences
+          locale.en.name: Infection sciences
+        /medical_device_risk_management_and_governance:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: medical_device_risk_management_and_governance
+          locale.en.name: Medical device risk management and governance
+        /medical_engineering:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: medical_engineering
+          locale.en.name: Medical engineering
+        /microbiology__healthcare_scientist_:
```

Comment:
Can we remove the trailing `_` from the name and key ?

Reply:
Going through and changing all of these now